src/Components/PreviewBlocks/PreviewBlocks.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import clinic_img from "./../../assets/Asset 26.svg";
+import gavel_img from "./../../assets/Asset 31.svg";
+import speaker_img from "./../../assets/Asset 28.svg";
+
+import "./PreviewBlocks.scss";
+
+const PreviewBlocks = () => {
+  return (
+    <>
+      <div className="preview">
+        <img src={clinic_img} alt="image-of-uterus" className="icon"></img>
+        <p>
+          Find a safe place to receive an abortion and avoid misinformation
+          centers. Learn what resources are available in states with total bans.
+        </p>
+        <Link to="/clinics">
+          <button>Find A Clinic</button>
+        </Link>
+      </div>
+      <div className="preview">
+        <img src={gavel_img} alt="image-of-gavel" className="icon"></img>
```

Comment:
alt tag could be either "gavel" or "icon of gavel", no need for hyphens since alt tags are for screen readers. 

(using an image in the alt tag is breaking the CI build, so just wanted to call out alternative language)